docs/kb/compression/fsicasestudy.md

```diff
@@ -3105,6 +3110,2757 @@ Data persisted via `set` may remain in the OS buffer cache before being written
+      <td id="T_9f355_row13_col0" class="data row13 col0" >2.4</td>
+      <td id="T_9f355_row13_col1" class="data row13 col1" >2.7</td>
+      <td id="T_9f355_row13_col2" class="data row13 col2" >2.2</td>
+      <td id="T_9f355_row13_col3" class="data row13 col3" >1.8</td>
+      <td id="T_9f355_row13_col4" class="data row13 col4" >2.1</td>
+      <td id="T_9f355_row13_col5" class="data row13 col5" >2.1</td>
+      <td id="T_9f355_row13_col6" class="data row13 col6" >2.0</td>
+      <td id="T_9f355_row13_col7" class="data row13 col7" >2.0</td>
+      <td id="T_9f355_row13_col8" class="data row13 col8" >1.6</td>
+      <td id="T_9f355_row13_col9" class="data row13 col9" >1.8</td>
+      <td id="T_9f355_row13_col10" class="data row13 col10" >2.0</td>
+      <td id="T_9f355_row13_col11" class="data row13 col11" >2.7</td>
+      <td id="T_9f355_row13_col12" class="data row13 col12" >2.2</td>
+      <td id="T_9f355_row13_col13" class="data row13 col13" >1.7</td>
+      <td id="T_9f355_row13_col14" class="data row13 col14" >2.3</td>
+      <td id="T_9f355_row13_col15" class="data row13 col15" >2.0</td>
+      <td id="T_9f355_row13_col16" class="data row13 col16" >2.4</td>
+    </tr>
+  </tbody>
+</table>
+
+Compression improves performance when large datasets are read from slow storage. Thus, it is **recommended for cold tiers** (rarely accessed data).
+
+# Summary
+
+For an optimal balance of cost and query performance, we recommend a tiered storage strategy
+
+   1. **Hot tier** should not employ compression to allow maximal ingestion rate and fastest queries. The hot tier should be located on fast storage. This tier typically stores a few weeks to month of data.
+   1. **The second tier** contains a high volume of less frequently queried data. We recommend using compression. If the goal is ultimate query speed, then `snappy` or `lz4` with level 5 or 6 are good choices. Choose `lz4` if the second priority is storage saving and use `snappy` if you would like the data migration process (from the hot tier) to be fast. If the storage space is limited and you would like to achieve a high compression ratio, then use 
+      * `zstd` level 10 for most columns.
+      * `gzip` level 5 for sequence number-like columns.
+      * Columns with parted attribute (e.g. `sym`) are exceptions and should not be compressed.
+   1. The **cold tier** contains high volume, rarely accessed data. It is typically placed on cheaper and slower storage, like object storage or HDD-backed solutions. We recommend using compression. If the second tier uses `lz4` or `snappy` then you might want to recompress the data with `zstd` to save more storage space.
+
+Not all tables need to be cut the same way. Frequently queried, "favourit" tables may reside in hot tier for longer period. On the contrary, there might be rarely queried columns even in a favourit table. **Make use of symbolic links to relocate column files to the preferred storage**.
```

Comment:
2 places that say 'favourit' . Not sure what this is, may have meant 'favourite'. May as well remove it completely, just leaving it as a 'frequently queried' table rather than additional terms.